### PR TITLE
Doc: Added the table of WAN validation

### PR DESCRIPTION
**Pull request by MaheshGSLAB**
## Change Summary

Added the table of WAN validation.

## Component(s) name

AVD Documentation

## Proposed changes

Added the table of WAN validation to WAN How-To guide

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md [hunk 1]
```diff
@@ -853,3 +853,14 @@ wan_virtual_topologies:
...
 ```
+
+### WAN Validation
+
+`eos_validate_state` is being enriched to support new tests for WAN designs. The tests listed below are validating WAN designs.
+
+| AVD Test Class | ANTA Test Class | Description |
+| -------------- | --------------- | ----------- |
+| AvdTestInterfacesState | VerifyInterfacesStatus | Validate the DPS interface status. |
+| AvdTestBGP | VerifyBGPSpecificPeers | Validate the different BGP Address Families Identifiers (AFI) and Subsequent Address Family Identifiers (SAFI) sessions state. |
```
Comment (gmuloc) [inline ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md]: Can you add which address families are checked for AutoVPN (path-selection) and CV Pathfinder (link-state / srte) + any other we check

File: ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md [hunk 2]
```diff
@@ -853,3 +853,14 @@ wan_virtual_topologies:
...
+| AvdTestIPSecurity | VerifySpecificIPSecConn | Validates the establishment of IP security connections for a peer within the default VRF. In its current state, the test validates only IPsec connections defined as static peers under the `router path-selection` section of the configuration. |
```
Comment (gmuloc) [inline ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md]: we are talking about WAN here so lets reword - same for the STUN one, you can remove "for WAN scenarios" as we are talking about WAN here
```suggestion
| AvdTestIPSecurity | VerifySpecificIPSecConn | Validates the establishment of IP security connections for each static peer under the `router path-selection` section of the configuration. |
```

File: ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md [hunk 3]
```diff
@@ -853,3 +853,14 @@ wan_virtual_topologies:
...
+| AvdTestStun | VerifyStunClient | Validates the presence of a STUN client translation for a given source IPv4 address and port for WAN scenarios. The list of expected translations for each device is built by searching local interfaces in each path-group. |
```
Comment (gmuloc) [inline ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md]: is there a plan to leverage the AVT tests at some point? https://anta.arista.com/stable/api/tests.avt/

Comment (MaheshGSLAB) [reply ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md]: Yes in next set of tasks will add AVT tests and DPS reachability test.

File: ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md (hunk 1 above)
Comment (MaheshGSLAB) [reply ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md]: No, not right now apart from path-selection, link-state and sr-te

File: ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md (hunk 2 above)
Comment (MaheshGSLAB) [reply ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md]: fix

File: ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md (hunk 3 above)
Comment (gmuloc) [reply ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md]: Thanks! Let's add a comment under the table that more tests are coming

**Review by gmuloc — APPROVED** [review]
LGTM - thanks!

File: ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md [hunk 4]
```diff
@@ -853,3 +853,16 @@ wan_virtual_topologies:
...
+`eos_validate_state` is being enriched to support new tests for WAN designs. The tests listed below are validating WAN designs.
+
+| AVD Test Class | ANTA Test Class | Description |
+| -------------- | --------------- | ----------- |
+| AvdTestInterfacesState | VerifyInterfacesStatus | Validate the DPS interface status. |
+| AvdTestBGP | VerifyBGPSpecificPeers | Validate the state of BGP Address Family sessions, including `Path-Selection` for AutoVPN, `Link-State` and `IPv4/IPv6 SR-TE` for CV Pathfinder. |
+| AvdTestIPSecurity | VerifySpecificIPSecConn | VValidates the establishment of IP security connections for each static peer under the `router path-selection` section of the configuration. |
+| AvdTestStun | VerifyStunClient | Validates the presence of a STUN client translation for a given source IPv4 address and port. The list of expected translations for each device is built by searching local interfaces in each path-group. |
+
+    More tests are coming soon.
```
Comment (gmuloc) [inline ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md]:
```suggestion
!!! note
    More WAN-related tests are available directly in ANTA and can be added using custom catalogs.
    They will be progressively added to `eos_validate_state`.
```

File: ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md [hunk 5]
```diff
@@ -853,3 +853,18 @@ wan_virtual_topologies:
...
+
+### WAN Validation
+
+`eos_validate_state` is being enriched to support new tests for WAN designs. The tests listed below are validating WAN designs.
+
+| AVD Test Class | ANTA Test Class | Description |
+| -------------- | --------------- | ----------- |
+| AvdTestInterfacesState | VerifyInterfacesStatus | Validate the DPS interface status. |
+| AvdTestBGP | VerifyBGPSpecificPeers | Validate the state of BGP Address Family sessions, including `Path-Selection` for AutoVPN, `Link-State` and `IPv4/IPv6 SR-TE` for CV Pathfinder. |
+| AvdTestIPSecurity | VerifySpecificIPSecConn | VValidates the establishment of IP security connections for each static peer under the `router path-selection` section of the configuration. |
```
Comment (carl-baillargeon) [inline ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md]:
```suggestion
| AvdTestIPSecurity | VerifySpecificIPSecConn | Validate the establishment of IP security connections for each static peer under the `router path-selection` section of the configuration. |
```

File: ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md [hunk 6]
```diff
@@ -853,3 +853,18 @@ wan_virtual_topologies:
...
+| AvdTestStun | VerifyStunClient | Validates the presence of a STUN client translation for a given source IPv4 address and port. The list of expected translations for each device is built by searching local interfaces in each path-group. |
```
Comment (carl-baillargeon) [inline ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md]:
```suggestion
| AvdTestStun | VerifyStunClient | Validate the presence of a STUN client translation for a given source IPv4 address and port. The list of expected translations for each device is built by searching local interfaces in each path-group. |
```

**Review by carl-baillargeon — APPROVED** [review]

**Pull request merged**